pkgs/tools/misc/lottieconverter/default.nix

```diff
@@ -1,34 +1,42 @@
-{ lib, stdenv, fetchFromGitHub, libpng, rlottie, zlib }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libpng
+, rlottie
+, giflib
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "LottieConverter";
```

Comment:
The pname attribute [must not](https://nixos.org/manual/nixpkgs/unstable/#chap-conventions) contain uppercase letters. Not sure if it is a breaking change.

Reply:
Maybe nix-env or others would depend on pname?

Reply:
Anyway l'd like to fix it.